File: examples/performance/objects/objects.php
<?php

class Box
{
    public $number;

    public function __construct($number)
    {
        $this->number = $number;
    }
}

for ($i = 0; $i < 100000000; $i++) { // 100_000_000
    $o = new Box($i);
}

File: examples/performance/objects/objects.pl
use strict;

package Box;
sub Box_data
{
    my $class = shift;
    my $self = {
               'number' => shift
               };

    bless $self, $class;
    return $self;
}

for (my $i = 0; $i < 100_000_000; $i++) {
    my $o = Box_data Box( $i );
}
